keep/api/models/db/migrations/versions/2024-12-23-18-49_4f8c4b185d5b.py

```diff
@@ -0,0 +1,42 @@
+"""Add is_provisioned column for DeduplicationRule table
```

Comment:
did you fix migration after merge from main? (fixed head)

Reply:
Yes

Reply:
I regenerated it